src/View/Helper/FormHelper.php

```diff
@@ -367,11 +367,16 @@ public function create($model = null, array $options = [])
         }
         unset($options['templates']);
 
-        $url = $this->_formUrl($context, $options);
-        $action = $this->Url->build($url);
-        unset($options['url'], $options['action'], $options['idPrefix']);
+        if ($options['url'] === false) {
+            $action = null;
+            $this->_lastAction = '';
```

Comment:
Yes I am aware of that and [stated](https://github.com/cakephp/cakephp/issues/6488#issuecomment-99184625) the same on related issue. Please read the discussion on #6488 for why people want this feature and why it won't be used with SecurityComponent.

We can throw an exception if `url` is set to false and `$this->request['_Token']` is not empty.